tests/ecosystem/upgrade/test_noobaa.py

```diff
@@ -1,6 +1,7 @@
 import logging
 
 import pytest
+import time
```

Comment:
its very minor, but time should be together with logging (built-in lib) and not with pytest(3rd party lib)

Reply:
>  its very minor, but time should be together with logging (built-in lib) and not with pytest(3rd party lib)

+1